microbench/src/main/java/io/netty5/microbench/buffer/BufferUtilBenchmark.java

```diff
@@ -58,9 +57,7 @@ public void setup() {
         buffer = direct? Unpooled.directBuffer(maxBytes) : Unpooled.buffer(maxBytes);
         wrapped = Unpooled.unreleasableBuffer(direct? Unpooled.directBuffer(maxBytes) : Unpooled.buffer(maxBytes));
         asciiSequence = new StringBuilder(length);
-        for (int i = 0; i < length; i++) {
-            asciiSequence.append('a');
-        }
+        asciiSequence.append("a".repeat(Math.max(0, length)));
         ascii = asciiSequence.toString();
```

Comment:
The `asciiSequence` can be removed as `String.repeat` can directly produce the `ascii` value.